hardhat.config.ts

```diff
@@ -22,7 +22,16 @@ task('accounts', 'Prints the list of accounts', async (taskArgs, hre) => {
 });
 
 const config: HardhatUserConfig = {
-  solidity: '0.8.4',
+  solidity: {
+    version: '0.8.4',
+    settings: {
+      outputSelection: {
+        '*': {
+          '*': ['storageLayout'],
```

Comment:
What is this needed for?

Reply:
This is required for using the smockit library 
https://smock.readthedocs.io/en/latest/getting-started.html